examples/hello/hello.js

```diff
@@ -1,13 +1,15 @@
-var button = new tabris.Button({
+const {Button, TextView} = require('tabris');
+
+const button = new Button({
   centerX: 0, top: 100,
   text: 'Show message'
 }).appendTo(tabris.ui.contentView);
```

Comment:
Avoid the global object `tabris`. Import `ui` and replace with `appendTo(ui.contentView)`.